_data/gaming.yml

```diff
@@ -18,7 +18,9 @@ websites:
       img: bdo.png
       twitter: BDO_News
```

Comment:
```suggestion
```

Reply:
When `tfa: Yes` social media links to tell them to add 2FA are no longer needed.